server/tserver/src/main/java/org/apache/accumulo/tserver/log/DfsLogger.java

```diff
@@ -558,7 +559,6 @@ public LoggerOperation defineTablet(CommitSession cs) throws IOException {
   private synchronized void write(LogFileKey key, LogFileValue value) throws IOException {
     key.write(encryptingLogFile);
     value.write(encryptingLogFile);
-    encryptingLogFile.flush();
```

Comment:
This moved from a synchronzied block to an unsynchronized block.   Could possibly be problematic, but I am not completely sure.  Depends a lot on the stream impl.

Reply:
Isn't this the same behavior of the WAL normally, when it is not encrypted?  Flushing/syncing in a separate unsychronized thread, while many threads write key values in a synchronized method?

Reply:
It's not quite the same.  What you described is for the lower level hdfs output stream.  This change makes the higher level encryption stream do that.

Reply:
I tested this with Uno and the encryption plugin enabled.  I was able to ingest 3 million rows and scan through the shell.  So I believe this works with our AES crypto impl.  I guess if someone discovered an impl that it doesn't work with, we can address it then.